typedmodels/models.py

```diff
@@ -205,15 +205,15 @@ def get_type_classes(subcls):
                     return list(cls._typedmodels_registry.values())
                 else:
                     return [
-                        cls._typedmodels_registry[k]
+                        cls._typedmodels_registry[k.id]
                         for k in subcls._typedmodels_subtypes
                     ]
 
             cls.get_type_classes = classmethod(get_type_classes)
 
             def get_types(subcls):
                 if subcls is cls:
-                    return list(cls._typedmodels_registry.keys())
+                    return ContentType.objects.get(id__in=list(cls._typedmodels_registry.keys()))
```

Comment:
This looks incorrect - `.get()` returns a single object but we want a list.

I'd suggest something like this instead:
```suggestion
                    return [ContentType.objects.get_for_id(pk) for pk in cls._typedmodels_registry.keys()]
```